### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by ACMCMC**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,4 +52,4 @@ As my Bitwarden instance is a critical part of my daily workflow and part of acc
 
 Your Bitwarden instance will go to sleep after 30 minutes of no activity. This should not be too bad of an issue due to the fact that you can maintain a local copy. However if you are adding, you may wish to have a cron job which polls your instance to keep it avaliable (read: Pingdom set to 15 minute intervals or any website status checker).
 
-The JawsDB instance comes with 5MB of storage space. I found this sufficient enough for my own personal backups even with 700+ entries, two orgs, and 4 members. You may find if you are attaching content, that you might exceed this but I suggest attach files in base64 encoded content to preserve portability.
\ No newline at end of file
+The JawsDB instance comes with 5MB of storage space. I found this sufficient enough for my own personal backups even with 700+ entries, two orgs, and 4 members. You may find if you are attaching content, that you might exceed this but I suggest attach files in base64 encoded content to preserve portability.
```
Comment (davidjameshowell) [inline README.md:55]: I don't see any changes here? Could you clarify the correction here?

Comment (ACMCMC) [reply README.md:55]: The change is on line 17 (simply -> simple). I'm not sure why Git is highlighting line 55 instead, but take a closer look at the changes view and you'll be able to see it :)

File: README.md [hunk 2]
```diff
@@ -52,4 +52,4 @@ As my Bitwarden instance is a critical part of my daily workflow and part of acc
 
 Your Bitwarden instance will go to sleep after 30 minutes of no activity. This should not be too bad of an issue due to the fact that you can maintain a local copy. However if you are adding, you may wish to have a cron job which polls your instance to keep it avaliable (read: Pingdom set to 15 minute intervals or any website status checker).
 
-The JawsDB instance comes with 5MB of storage space. I found this sufficient enough for my own personal backups even with 700+ entries, two orgs, and 4 members. You may find if you are attaching content, that you might exceed this but I suggest attach files in base64 encoded content to preserve portability.
```
Comment (ACMCMC) [inline README.md:55]: These lines are the same

File: README.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ Deploy Bitwarden_rs in Heroku for free via Github1
 
 ## Usage
 
-Usage is simply, fast, and user friendly!
```
Comment (ACMCMC) [inline README.md:17]: The change is here

File: README.md (hunk 1 above)
Comment (ACMCMC) [reply README.md:55]: I'm not sure why Git is highlighting that line, but indeed, the change is not there :)

**Pull request merged**